validator/src/main.rs

```diff
@@ -3,7 +3,7 @@ use clap::{
     ArgMatches,
```

Comment:
`validator/src/main.rs` will randomly run through all the provided entrypoints and only hard fail if they are all unreachable